plugin/latexlivepreview.vim

```diff
@@ -184,14 +184,23 @@ EEOOFF
                         \ l:tmp_root_dir . '/' . bib_fn)                                " TODO: may fail if same bibfile names in different dirs
         endfor
 
+        if exists('g:livepreview_biber') && g:livepreview_biber > 0
```

Comment:
Other configuration flags ( `engine`, `previewer` ) can set the binary directly, we could try to do something similar for bibliography 

```suggestion
        if (s:bibengine == 'biber')
```

---

We should also set this in the `Initialize` function ( found this in https://stackoverflow.com/a/33775128, to be tested though )

```vim
    " Get the bib engine
    let s:bibengine = get(g:, 'livepreview_bibengine', 'bibtex')
```

https://github.com/xuhdev/vim-latex-live-preview/blob/855c30915547c7e9c19b7e6efb528730c071d2e8/plugin/latexlivepreview.vim#L230-L252